pigpio.js

```diff
@@ -90,7 +90,12 @@ class Gpio extends EventEmitter {
     pigpio.gpioHardwarePWM(this.gpio, +frequency, +dutyCycle);
     return this;
   }
-
+  tick() {
```

Comment:
Why is this an instance method in class Gpio?
It should be a module level function like `hardwareRevision`,

Reply:
Moved at module level with f716731